.gitmodules

```diff
@@ -1,4 +1,4 @@
-[submodule "notebooks"]
-	path = notebooks
+[submodule "_notebooks"]
+	path = _notebooks
```

Comment:
what does this do?

Reply:
git does and eventually some IDEs are not happy when you switch between master (or PRs created from before refactored) and release branch as once it is a folder other time it is a reference to submodule... also a little visual diff that it is not a normal folder :rabbit: